code/modules/mob/living/basic/jungle/venus_human_trap.dm

```diff
@@ -120,13 +120,28 @@
  * The result of a kudzu flower bud, these enemies use vines to drag prey close to them for attack.
  *
  * A carnivorious plant which uses vines to catch and ensnare prey.  Spawns from kudzu flower buds.
- * Each one has a maximum of four vines, which can be attached to a variety of things.  Carbons are stunned when a vine is attached to them, and movable entities are pulled closer over time.
+ * Carbons are stunned when a vine is attached to them, and movable entities are pulled closer over time.
  * Attempting to attach a vine to something with a vine already attached to it will pull all movable targets closer on command.
  * Once the prey is in melee range, melee attacks from the venus human trap heals itself for 10% of its max health, assuming the target is alive.
  * Akin to certain spiders, venus human traps can also be possessed and controlled by ghosts.
  *
  */
-/mob/living/simple_animal/hostile/venus_human_trap
+
+/datum/ai_controller/basic_controller/human_trap
```

Comment:
Move this back to the bottom of the file, or move the big comment block.
Having it here now means all that documentation is on the AI controller, not the mob.

Reply:
(Do these mobs even ever exist with AI?)

Reply:
no i dont think they do

Reply:
I guess it isn't harmful in case someone wants to stick them on a ruin (although they'll probably just die from lack of vines).